package/gluon-config-mode-outdoor/i18n/de.po

```diff
@@ -0,0 +1,10 @@
+msgid ""
+"Please enable this option in case the node is to be installed outdoors "
+"to comply with local frequency regulations."
+msgstr ""
+"Wenn der Knoten im Freien aufgestellt werden soll, dann aktiviere bitte "
+"diese Option um den örtlichen Frequenzbestimmungen zu entsprechen."
+
+msgid "Node will be installed outdoors"
+msgstr "Knoten wird im Außenbereich betrieben"
+
```

Comment:
Sorry I don't mean a newline like that. In the version before there was the line ending broken. Now the line ending seems ok, but the "newline" should be remove.